lib/va_profile/profile/v3/configuration.rb

```diff
@@ -31,7 +31,7 @@ def service_name
         ##
         # @return [Faraday::Response] response from POST request
         #
-        def post(path, body = {})
+        def submit(path, body = {})
```

Comment:
Suggestion: Instead of changing the method name, add `alias_method :submit, :post`.

Reply:
👍 Fixed.